### DISCUSSION TITLE
broken wiki link: rofi-firefox

### DISCUSSION THREAD

**Discussion by CouldBeThis** [davatorium/rofi#1697] [(Possible) Issues] [answered]
Hi, I was just looking through [wiki/User-scripts](https://github.com/davatorium/rofi/wiki/User-scripts) and noticed that the link to [rofi-firefox](https://github.com/m0lese/configuration-files/blob/prime/scripts/rofi/rofi-firefox) is dead and the user who owned the repo no longer exists at all. 

[Searching](https://github.com/search?q=rofi-firefox) on github finds multiple repos of the same name but I do not know which if any of them are the one that was linked to before. So not sure how to fix the link.

**Comment by DaveDavenport** [comment, marked as answer]
Remove the broken link?
